heartbeat/galera

```diff
@@ -215,6 +218,7 @@ mapping in option cluster_host_map.
 <content type="string" default=""/>
 </parameter>
 
+
```

Comment:
Why are you adding empty lines to the longdesc?

Reply:
likely an editor artifact.    I'm also looking for reaction to the overall concept of embedding SSH commands into a resource agent and communicating with other clusters (hence WIP).